app/views/user_preferences/edit.html.erb

```diff
@@ -1 +1,25 @@
-<%raise%>
\ No newline at end of file
+
+<div class="container-sm">
+  <h3>Edit your profile</h3>
+  <br>
+  <%= simple_form_for @user_preference do |f| %>
+  <%= f.input :name, label: "Name"%>
+  <br>
+  <br>
+  <%= f.input :city, label: "Your location"%>
+  <br>
+  <br>
+  <%= f.input :avatar, label: "Choose your avatar"%>
```

Comment:
Needs to limit input otherwise user can input anything anything causing unexpected behaviour

Reply:
`  <%= f.input :avatar, as: :radio_buttons, collection: [['0', 'female'], ['1', 'male']], label_method: :second, value_method: :first %>
`